egs/babel/s5b/local/train_g2p.sh

```diff
@@ -55,7 +55,7 @@ fi
 if [ ! -z $icu_transform ] ; then
   paste \
     <(cat $lexicon | awk '{print $1}' | uconv -f $encoding -t $encoding -x "$icu_transform") \
-    <(cat $lexicon | sed $'s/^[^ \t][^ \t]*[ \t]//g') \
+    <(cat $lexicon | perl -ape 's/^[^ \t][^ \t]*[ \t]//g;') \
```

Comment:
because this regex is more complicated you should check with some example input that it does what you think it does (different progrmas have different conventions).